service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/IllogicalMessageComponentTest.java

```diff
@@ -139,6 +140,7 @@ public void When_ContinueReceivedOutOfOrderExtractCoreNotSent_Expect_ErrorThrown
     }
 
     @Test
+    @Disabled
     public void When_AcknowledgementReceivedOutOfOrderAcknowledgmentNotSent_Expect_ErrorThrown() {
```

Comment:
Is this test supposed to be disabled?

Reply:
I'll add a TODO item to change that in the future